README.md

```diff
@@ -68,11 +47,11 @@ Make sure that you've committed your changes first, because Middleman works with
 bundle install
 bundle exec middleman init tmp/test-run -T file://$(pwd)
 cd tmp/test-run
-bundle exec middleman server
+bundle install
+bundle exec middleman build --verbose
```

Comment:
Any reason we switch from server to build here? didn't realise that ran a server

Reply:
Nope, it doesn't. Fixed in https://github.com/alphagov/tech-docs-template/pull/144/commits/4603cb35ffc932c335dec882a8e9d04236644c18.